src/documents/IoCSS.md

```diff
@@ -4,13 +4,18 @@ layout: default
 category: Infusion
 ---
 
-The Infusion Inversion of Control (IoC) system includes a simple method for distributing options from top-level components through the component tree. This method is dubbed "IoCSS" because it uses a CSS-like syntax.
+The Infusion Inversion of Control (IoC) system includes a simple method for distributing options from any components to any others in the component tree. This system uses 
+a special block named `distributeOptions` which is supported in the options of every Infusion component. The key addressing scheme needed to make this system work &#8212; the
+means for determining where the target of the distribution is &#8212; is dubbed "IoCSS" because it uses a syntax and idiom very similar to that used in [CSS selectors](https://en.wikipedia.org/wiki/Cascading_Style_Sheets#Selector).
 
-## distributeOptions and IoCSS: Downward-Matching CSS-Like Context Selectors For Options Forwarding ##
+## `distributeOptions and IoCSS`: Downward-Matching CSS-Like Context Selectors For Options Forwarding ##
 
-The `distributeOptions` option is a top-level block supported by every IoC-configured component. It specifies how options should be distributed to subcomponents further down the tree.
+The `distributeOptions` option is a top-level block supported by every Infusion component. It specifies how options should be distributed to components elsewhere in the tree. A typical, conservative
+usage would have a component distribute only *downwards* to its own subcomponents, but it is also possible to use the considerable power of `distributeOptions` to distribute to *any component
+anywhere* in the (global) component tree - naturally such power should be used with extreme discretion since it could very easily reduce a design to chaos. However, in many situations, for example
+when writing powerful authoring tools, this kind of power is completely indispensible.
 
-## Background ##
+## Background for `distributeOptions`##
```

Comment:
I wonder if most of this background information would be better placed after the technical information that currently follows it. Otherwise, a reader who just needs to look up the formats needs to wade through this information first.